### PR TITLE
Checkbox: Checkmark Visibility Token

### PR DESCRIPTION
**Pull request by samuelfreiberg**
Received a request from Docs team to allow support to show Checkmark on Hover. This token allows a user to implement that. 

Checkbox Docs modified, here's the PR: [Checkbox Updated Docs](https://github.com/microsoft/fluent-site/pull/55)

### CODE REVIEW

**Comment by PPatBoyd** [comment]
change files needed (failed PR build)

File: docs/content/windows/components/Checkbox.mdx [hunk 1]
```diff
@@ -24,6 +24,7 @@ Checkbox supports the following tokens:
   2. checkboxBorderColor – This changes the border color of the Checkbox.
   3. checkmarkColor – This changes the color of Checkmark.
   4. borderRadius - This changes the border radius of the Checkbox (use this to create a circular checkbox)
+  5. checkmarkVisibility - Allows you to change Checkmark visibility - 1 is visible. More info in Examples below.
```
Comment (PPatBoyd) [inline docs/content/windows/components/Checkbox.mdx:27]: j.w. are there other expected scenarios besides the hover checkbox?  does the hover checkbox show the check in different colors for hover vs. committed vs committed + hover?

Also these docs changes should go into the microsoft/fluentui-site repo; I would link the PR there from this PR description

Comment (samuelfreiberg) [reply docs/content/windows/components/Checkbox.mdx]: This token allows you to change the Opacity of the Checkmark, there's another token that allows you to change the color of the token

Comment (samuelfreiberg) [reply docs/content/windows/components/Checkbox.mdx]: Linked the Docs in this PRs description

**Pull request merged**